rfcs/experimental/kernel_templates/README.md

```diff
@@ -290,6 +290,20 @@ The `kernel_param` targets a single kernel, and currently, only one instance can
 Currently, it is unclear if we should defferentiate algorithms
 which rely on compiler extensions.
 
+### Kernel Templates as a Backend for Algorithms with Standard Interfaces
+
+It should be evaluated whether the proposed algorithms can serve as a backend for oneDPL algorithms
+that use the standard C++ interfaces and device execution policies.
+
+It will affect which data-passing mechanisms,
+as described in [Abstract Algorithm Signature](abstract-algorithm-signature) section, are allowed.
+In particular, it should be investigated if supporting these entities is necessary and
+whether they can be handled by an additional thin layer between
+the algorithms with standard interfaces and Kernel Templates:
+- `oneapi::dpl::begin` and `oneapi::dpl::end`, which mimic iterators to pass `sycl::buffer` to algorithms accepting iterators.
+- Iterators to a host-allocated `std::vector` as described in
+  [Pass Data to Algorithms > Use std::vector](https://github.com/uxlfoundation/oneDPL/blob/release/2022.9.0/documentation/library_guide/parallel_api/pass_data_algorithms.rst#use-stdvector)
```

Comment:
At glance, it does not seem necessary for KT, even if it serves as the implementation backend for the "regular" algorithms. The necessary data copying/transferring to the device can and likely should be performed before KT is used.

Reply:
I've removed this paragraph in the KT as a backed sub-section, and updated the main description on data passing mechanisms. See https://github.com/uxlfoundation/oneDPL/pull/2301/commits/5001928fdddcaf30cff26418e1bbb6fb59e0336f.